src/components/icon/icons/wallet.jsx

```diff
@@ -0,0 +1,51 @@
+            id="Shape"
+          />
+          <path
+            d="M13.8947368,14.2662539 L10.5263158,14.2662539 C10.2938947,14.2662539 10.1052632,14.0887183 10.1052632,13.869969 C10.1052632,13.6512198 10.2938947,13.4736842 10.5263158,13.4736842 L13.8947368,13.4736842 C14.1271579,13.4736842 14.3157895,13.6512198 14.3157895,13.869969 C14.3157895,14.0887183 14.1271579,14.2662539 13.8947368,14.2662539 Z"
+            id="Path"
+          />
+          <path
+            d="M7.41092093,4.06540416 C7.34287782,3.99296482 7.22538261,3.98596585 7.14841582,4.05000642 C6.91082266,4.24737737 6.60518639,4.35621136 6.28765187,4.35621136 C5.59680979,4.35621136 5.03015569,3.84668635 4.98888364,3.20628059 L5.04093848,3.25527338 C5.07737686,3.28956834 5.12496986,3.30636586 5.17256285,3.30636586 C5.22015585,3.30636586 5.26774885,3.28921839 5.30418723,3.25527338 C5.37669219,3.18703343 5.37669219,3.07609975 5.30418723,3.0078598 L4.93236695,2.6579113 C4.859862,2.58967134 4.74199497,2.58967134 4.66949002,2.6579113 L4.29766974,3.0078598 C4.22516478,3.07609975 4.22516478,3.18703343 4.29766974,3.25527338 C4.37017469,3.32351334 4.48804172,3.32351334 4.56054668,3.25527338 L4.61669154,3.20243116 C4.65610449,4.03810817 5.39082136,4.7065098 6.28802369,4.7065098 C6.69591054,4.7065098 7.08892457,4.5665304 7.39456084,4.31281774 C7.47152764,4.24877717 7.47896404,4.13819344 7.41092093,4.0657541 L7.41092093,4.06540416 Z"
+            id="Path"
+          />
+          <path
+            d="M8.27837764,3.00750985 C8.20587269,2.93926989 8.08800566,2.93926989 8.01550071,3.00750985 L7.95935584,3.06035207 C7.91994289,2.22467506 7.18522602,1.55627343 6.28802369,1.55627343 C5.88013685,1.55627343 5.48712281,1.69590288 5.18148654,1.94996549 C5.10451974,2.01400606 5.09708334,2.12458979 5.16512645,2.19702913 C5.23316956,2.26946847 5.35066477,2.27646744 5.42763157,2.21242686 C5.66522472,2.01505591 5.97086099,1.90622193 6.28839551,1.90622193 C6.97923759,1.90622193 7.54589169,2.41574694 7.58716375,3.05615269 L7.53510891,3.0071599 C7.46260395,2.93891994 7.34473692,2.93891994 7.27223197,3.0071599 C7.19972701,3.07539986 7.19972701,3.18633353 7.27223197,3.25457349 L7.64405225,3.60452198 C7.68049064,3.63881694 7.72808363,3.65561447 7.77567663,3.65561447 C7.82326962,3.65561447 7.87086262,3.63846699 7.90730101,3.60452198 L8.27912128,3.25457349 C8.35162624,3.18633353 8.35162624,3.07539986 8.27912128,3.0071599 L8.27837764,3.00750985 Z"
+            id="Path"
```

Comment:
Let's remove ids from svg

Reply:
I think that this is a problem for every svg here, sadly 😢 So maybe a codemod would be fitting?